### PR TITLE
Complete NYT Webpage Article

### PR DESCRIPTION
**Pull request by kensayo**
**Pull Request second into main**

- Navbar Menu
- Header article image and title
- Full content in the main article is completed
- Aside Editors Link
- Footer sections with suggestions
- Footer with navigation to the main page
- Navbar at bottom

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,815 @@
...
+                </a>
+                <a class="nav-link margin-menu soft-black" href="#">
+                    <i class="fa fa-search"></i>
+                </a>
+                <a class="nav-link margin-menu" href="#">
+                    <h3 class="nyt-f fw-700 black">SPACE & COSMOS</h3>
+                </a>
+            </div>
+            <div class="nav-container">
+                <img src="./img/nyt-logo.png" alt="NYT Logo">
```
Comment (teekaytech) [inline index.html]: The NY Times logo should be linked. i.e. use the `<a>` tag around the logo. (<a href="https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0" rel="nofollow noreferrer noopener" target="_blank">link to page</a>)

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @kensayo :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- Linter checks are running/passing successfully :100: 
- Deployment working perfectly :+1: 
- Your Readme is professional :ok_hand: 
- Nice implementation of the `Editor's Picks` and `The Theory Of Inflation` sections :1st_place_medal: 
- You've paid so much attention to the little details of this project :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Update your Readme file: 
- Add a screenshot of your program to the readme.
- You should highlight the steps involved in cloning and using your project in the `Getting Started` section.
- [x] Kindly improve the summary of your PR. You're expected to summarize all features you've worked on in this PR. A message like `Pull for the main branch for NYT page` is not descriptive enough. 
- [x] Also, PR title like `Merge into main branch` does not convey much meaning to the PR. Titles like `NYT Article`, `NYT Article Webpage`, `NYT Article Project`, or any other befitting title may be used. Kindly change the PR title.


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html (hunk 1 above)
Comment (kensayo) [reply index.html]: Ready, thanks.

**Review by christianotieno — APPROVED** [review]
## Project Approved 🍾 
Hello @kensayo 👋🏾 
Congratulations. The project is now approved. 👏🏾 
All the very best in your next tasks 🥇 

Happy coding 💻 
Christian Otieno (CR)

**Pull request merged**